nodejs/src/tensor_helper.cc

```diff
@@ -232,14 +232,15 @@ Napi::Value OrtValueToNapiValue(Napi::Env env, Ort::Value &value) {
     auto stringArray = Napi::Array::New(env, size);
     if (size > 0) {
       auto tempBufferLength = value.GetStringTensorDataLength();
-      auto tempBuffer = std::make_unique<char[]>(tempBufferLength);
+      // create buffer of length (tempBufferLength + 1) to make sure `&tempBuffer[0]` is always valid
+      std::vector<char> tempBuffer(tempBufferLength + 1);
```

Comment:
why not std::string?

Reply:
I am not sure if std::string handles the buffer correctly, as it uses `\0` as the last char of the string

Reply:
It should be ok to use std::string as long as we do not call c_str().